leakcanary-object-watcher-android/src/main/java/leakcanary/AppWatcher.kt

```diff
@@ -43,6 +43,13 @@ object AppWatcher {
      */
     val watchFragmentViews: Boolean = true,
 
+    /**
+     * Whether AppWatcher should automatically watch cleared view model instances.
```

Comment:
To prevent ambiguity with `MVVM` term I'd suggest using `ViewModel` term instead of `view model` here. That's what Google call them inside `lifecycle-viewmodel` javadocs as well